src/main/java/io/codelex/classesandobjects/practice/videostore/Video.java

```diff
@@ -0,0 +1,38 @@
+package io.codelex.classesandobjects.practice.videostore;
+
+public class Video {
+    private String title;
+    private boolean checkedOut;
+    private int userRating;
+    private int ratingCount = 0;
+
+
+    public Video(String title) {
+        this.title = title;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void isCheckedOut() {
```

Comment:
šis nosauums būtu labāks boolean metodei
izņemšanai un atgriešanai `checkOut()` `return()` piemēram

Reply:
Pārveidoju šo metodi un arī is Returned() par boolean un noņēmu checkedOutStatus() metodi. Tālāk šīs metodes attiecīgi izmantotas videoStore klasē, lai pārbaudītu vai klientam konkrētais video ir pieejams. Princips nemainās, bet viss ir īsāk un saprotamāk tagad.